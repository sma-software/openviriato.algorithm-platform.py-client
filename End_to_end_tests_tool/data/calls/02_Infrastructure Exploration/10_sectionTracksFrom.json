{
    "restResource": "/section-tracks",
    "httpMethod": "GET",
    "urlParameters": null,
    "queryParameters": [
        {
            "key": "FromNodeID",
            "value": "161",
            "optional": true
        }
    ],
    "requestBody": null,
    "CSharpMethodName": "GetSectionTracksFrom(fromNodeID)",
    "CSharpMethodParameterTypes": [ "System.Int64" ],
    "subcategory": "Get Section Tracks For Node"
}
