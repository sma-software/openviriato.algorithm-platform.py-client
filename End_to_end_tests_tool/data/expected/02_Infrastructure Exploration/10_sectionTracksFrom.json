[
  {
    "id": 885,
    "code": "838",
    "sectionCode": "61010",
    "weight": 37040,
    "debugString": "sectiontrack:s_61010 n_85AR 838"
  },
  {
    "id": 886,
    "code": "2",
    "sectionCode": "61010",
    "weight": 37040,
    "debugString": "sectiontrack:s_61010 n_85AR 2"
  },
  {
    "id": 887,
    "code": "835",
    "sectionCode": "61010",
    "weight": 20910,
    "debugString": "sectiontrack:s_61010 n_85JE 835"
  }
]